test/integrations/destinations/google_adwords_enhanced_conversions/dataDelivery/business.ts

```diff
@@ -0,0 +1,157 @@
+};
+
+const params = {
+  event: 'Product Added',
+  customerId: '1234567899',
+  destination: 'google_adwords_enhanced_conversions',
+};
+
+const validRequestPaylod = {
+  partialFailure: true,
+  conversionAdjustments: [
+    {
+      gclidDateTimePair: {
+        gclid: 'gclid1234',
+        conversionDateTime: '2022-01-01 12:32:45-08:00',
+      },
+      restatementValue: {
+        adjustedValue: 10,
+        currency: 'INR',
+      },
+      order_id: '10000',
+      adjustmentDateTime: '2022-01-01 12:32:45-08:00',
+      userAgent:
+        'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36',
+      userIdentifiers: [
+        {
+          addressInfo: {
+            hashedFirstName: 'a8cfcd74832004951b4408cdb0a5dbcd8c7e52d43f7fe244bf720582e05241da',
+            hashedLastName: '1c574b17eefa532b6d61c963550a82d2d3dfca4a7fb69e183374cfafd5328ee4',
+            state: 'UK',
+            city: 'London',
+            hashedStreetAddress: '9a4d2e50828448f137f119a3ebdbbbab8d6731234a67595fdbfeb2a2315dd550',
+          },
+        },
+      ],
+      adjustmentType: 'ENHANCEMENT',
+    },
+  ],
+};
+
+const commonRequestParameters = {
+  headers,
+  params,
+  JSON: validRequestPaylod,
+};
+
+export const testScenariosForV0API = [
+  {
+    id: 'gaec_v0_scenario_1',
+    name: 'google_adwords_enhanced_conversions',
+    description:
+      '[Proxy v0 API] :: Test for a valid request with a successful 200 response from the destination',
+    successCriteria: 'Should return 200 with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          ...commonRequestParameters,
+          endpoint:
+            'https://googleads.googleapis.com/v15/customers/1234567899:uploadConversionAdjustments',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            destinationResponse: {
+              response: [
+                {
+                  results: [
+                    {
+                      adjustmentDateTime: '2021-01-01 12:32:45-08:00',
+                      adjustmentType: 'ENHANCEMENT',
+                      conversionAction: 'customers/7693729833/conversionActions/874224905',
+                      gclidDateTimePair: {
+                        conversionDateTime: '2021-01-01 12:32:45-08:00',
+                        gclid: '1234',
+                      },
+                      orderId: '12345',
+                    },
+                  ],
+                },
+              ],
+              status: 200,
+            },
+            message: 'Request Processed Successfully',
+            status: 200,
+          },
+        },
+      },
+    },
+  },
+];
+
+export const testScenariosForV1API: ProxyV1TestData[] = [
```

Comment:
Can we add a scenario where more than 1 conversion data is delivered and it partially succeeds

Reply:
addressed